api/include/opentelemetry/trace/span.h

```diff
@@ -29,6 +29,16 @@ enum class SpanKind
   kProducer,
   kConsumer,
 };
+
+// StatusCode - Represents the canonical set of status codes of a finished Span.
+
+enum class StatusCode
+{
+  Unset,  // default status
+  Ok,     // Operation has completed successfully.
+  Error   // The operation contains an error
```

Comment:
For enum values: we should prefix them with a `k`, according to the code style guide.

Reply:
Thank you. Have done the change